lib/components/Button/Button.stories.tsx

```diff
@@ -1,16 +1,17 @@
+import type { Meta, StoryObj } from 'storybook-solidjs'
+import { default as ButtonComponent } from './Button'
 import React from 'react'
-import { ComponentStory, ComponentMeta } from '@storybook/react'
 
-import  { default as ButtonComponent } from "./Button"
-
-export default {
-  title: "Components/Button",
-  component: ButtonComponent,
-} as ComponentMeta<typeof ButtonComponent>
+const meta: Meta<typeof ButtonComponent> = {
+  component: ButtonComponent
+}
 
-const Template: ComponentStory<typeof ButtonComponent> = args => <ButtonComponent {...args} />
+export default meta
+type Story = StoryObj<typeof ButtonComponent>
 
-export const Button = Template.bind({})
-Button.args = {
-  children: 'Primary'
+export const Default: Story = {
+  args: {
+    children: 'Primary'
+  },
+  render: (args: object) => <ButtonComponent {...args}>Primary</ButtonComponent>
```

Comment:
The test text(`Close`) doesn't seem to appear, since we always have `Primary` here
![Screenshot 2024-07-22 at 15 52 17](https://github.com/user-attachments/assets/4ce135b3-2d46-4f37-8c96-6d566161ee3f)